src/object/spawnpoint.hpp

```diff
@@ -25,6 +25,11 @@
 class ReaderMapping;
 class DrawingContext;
 
+/**
+* The spawnpoint that gets activated at the start of a game session)
+*/
+extern std::string DEFAULT_SPAWNPOINT_NAME;
```

Comment:
Maybe these constants would be more fit in `supertux/constants.hpp`? You wouldn't need to `#include` the whole `SpawnPoint` or `Sector` object to use any of them.

Also, declare them as `const`, since they're not meant to be modified.

Reply:
I don't think `const` applies to this, since it's not bound to a class.

Reply:
Not sure what you mean. I mean making the variable `const`.

Reply:
Yes of course. I'm stupid.